### PR TITLE
Add footer and privacy policy

### PR DESCRIPTION
**Pull request by HenryHins**
- Add footer
- Add privacy policy page
connects issue #9

### CODE REVIEW

File: content/policy/PolicyController.js [hunk 1]
```diff
@@ -18,13 +18,48 @@ function($scope, $anchorScroll, $routeParams, loginStatusProvider, eventHandler,
 	$scope.closeAlert = function(){ $scope.alert = null; };
 	$scope.UserAuthenticated = false;
 	$scope.links = [];
+	
 	function SetupUser() {
 		return loginStatusProvider.validateAuthenticationPromise()
```
Comment (wparad) [inline content/policy/PolicyController.js]: The only thing we need to do inside the validateAuthenticationPromise.then(function()) is to call
```
$scope.$apply(function() { $scope.UserAuthenticated = true; });
```
everything else doesn't apply to this screen, right?

**Review by wparad — CHANGES_REQUESTED** [review]
Looks great, just remove the extra code from the policy controller and I'll approve the PR.
Thanks!

File: content/policy/PolicyController.js (hunk 1 above)
Comment (HenryHins) [reply content/policy/PolicyController.js]: Alright! I've updated the PR

**Review by wparad — APPROVED** [review]

**Comment by wparad** [comment]
fixes issue #9

**Pull request merged**